### PR TITLE
Add python compiling to Tox tests

### PR DESCRIPTION
**Pull request by benjaoming**
### Summary

Adding this for 0.8, because the fix is small, but the bug is interesting enough.

This directly affects some hard-to-maintain patches in the debian installer, removing current python2-only syntax inside the supposed-ly py2+3 codebase.

Running for current env's Python version

WIP: The tests should reveal that `kathjax.py` shouldn't be there.

### Reviewer guidance

n/a

### References

#3152

----

### Contributor Checklist

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] Contributor has fully tested the PR manually
- [x] Screenshots of any front-end changes are in the PR description
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')

### Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] Reviewer has fully tested the PR manually
- [ ] PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- [ ] External dependencies files were updated (`yarn` and `pip`)
- [ ] Documentation is updated
- [ ] Link to diff of internal dependency change is included
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] Contributor is in AUTHORS.rst

### CODE REVIEW

**Comment by benjaoming** [comment]
> And are you using the fail+fix strategy so that there will be another commit to fix the errors in the current tests?

Yup, precisely! And I hope this fixed it! :)

**Comment by lyw07** [comment]
@benjaoming Thank you for the clarification! I only have a question about removing `kathjax.py`. Do you think it's safe for us to remove it? I saw that the file is included in the `mathjax` github repository: https://github.com/Khan/khan-MathJax/blob/master/kathjax.py, but I'm not familiar with `mathjax` or `kolibri_exercise_perseus_plugin` so I'm not sure what `kathjax.py` does here. Could you explain a bit? Thank you!

**Comment by benjaoming** [comment]
@lyw07 it's ready! :)

**Comment by lyw07** [comment]
@benjaoming Thank you! One more thing is that when you have time, could you explain the usage of `kathjax.py` if you can take a look at my comment above? That would be super helpful for me to review! Thanks! :)

**Comment by benjaoming** [comment]
@lyw07 ah yes, okay this file is something that they use for some kind of build task in Perseus. I don't think it's even supposed to be distributed since it's a .py file in a directory full of .js files. See: https://github.com/Khan/khan-MathJax/blob/master/kathjax.py
